launchpod/angular-launchpod/backend/src/main/java/com/google/launchpod/servlets/DataServlet.java

```diff
@@ -14,18 +14,36 @@
 
 package com.google.launchpod.servlets;
 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that returns some example content. */
-@WebServlet("/data")
+@WebServlet("/load-result")
 public class DataServlet extends HttpServlet {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("launchpod-step18-2020.appspot.com");
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String title = request.getParameter("title");
+    String mp3Link = request.getParameter("mp3Link");
+
+    // Create entity with the comment from the input form.
+    Entity feedEntity = new Entity("Feed");
+    feedEntity.setProperty("title", title);
+    feedEntity.setProperty("mp3Link", mp3Link);
+
+    // Add the entity to the datastore.
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    datastore.put(feedEntity);
```

Comment:
i don't think this will be the final implementation right? I think it might be better just to leave this method empty (that way a post request will return success). we can add a TODO comment to fill in the method later

Reply:
Yes, this servlet is just a stand-in that I was using to make sure post and get requests worked properly. I have changed the doPost to an empty method.